src/genie/libs/parser/asa/show_bgp.py

```diff
@@ -0,0 +1,523 @@
+                nbr_af_dict['as'] = neighbor_as
+                nbr_af_dict['msg_rcvd'] = int(m.groupdict()['msg_rcvd'])
+                nbr_af_dict['msg_sent'] = int(m.groupdict()['msg_sent'])
+                nbr_af_dict['tbl_ver'] = int(m.groupdict()['tbl_ver'])
+                nbr_af_dict['input_queue'] = int(m.groupdict()['inq'])
+                nbr_af_dict['output_queue'] = int(m.groupdict()['outq'])
+                nbr_af_dict['up_down'] = str(m.groupdict()['up_down'])
+                nbr_af_dict['state_pfxrcd'] = str(m.groupdict()['state'])
+                nbr_af_dict['route_identifier'] = route_identifier
+                nbr_af_dict['local_as'] = local_as
+                nbr_af_dict['bgp_table_version'] = bgp_table_version
+                nbr_af_dict['routing_table_version'] = routing_table_version
+
+                try:
+                # Assign variables
+                    if attribute_entries:
+                        nbr_af_dict['attribute_entries'] = attribute_entries
+                    if num_prefix_entries:
+                        nbr_af_dict['prefixes'] = {}
+                        nbr_af_dict['prefixes']['total_entries'] = num_prefix_entries
+                        nbr_af_dict['prefixes']['memory_usage'] = num_memory_usage
+
+                    if path_total_entries:
+                        nbr_af_dict['path'] = {}
+                        nbr_af_dict['path']['total_entries'] = path_total_entries
+                        nbr_af_dict['path']['memory_usage'] = path_memory_usage
+
+                    if total_memory:
+                        nbr_af_dict['total_memory'] = total_memory
+
+                    if activity_prefixes:
+                        nbr_af_dict['activity_prefixes'] = activity_prefixes
+
+                    if activity_paths:
+                        nbr_af_dict['activity_paths'] = activity_paths
+
+                    if scan_interval:
+                        nbr_af_dict['scan_interval'] = int(scan_interval)
+
+                    if len(cache_dict):
+                        nbr_af_dict['cache_entries'] = cache_dict
+
+                    if len(entries_dict):
+                        nbr_af_dict['entries'] = entries_dict
+
+                    if num_community_entries:
+                        nbr_af_dict['community_entries'] = {}
+                        nbr_af_dict['community_entries']['total_entries'] = num_community_entries
+                        nbr_af_dict['community_entries']['memory_usage'] = community_memory_usage
+                except Exception:
+                    pass
+            else:
+                # when neighbor info break down to 2 lines.
+                #  Neighbor        V           AS MsgRcvd MsgSent   TblVer  InQ OutQ Up/Down  State/PfxRcd
+                #  2001:DB8:20:4:6::6
+                #           4          400      67      73       66    0    0 01:03:11        5
+                m = p10.match(line)
+                if m :
```

Comment:
I don't think this is necessary to put under an `else`. If `p9` isn't matched it should go to `p10` by default. If `p9` is matched you can simply have a `continue` at the end of it so `p10` is never reached